stdlib/public/SDK/AppKit/AppKit.swift

```diff
@@ -141,3 +141,10 @@ extension NSWindowController {
     self.dismissController(sender)
   }
 }
+
+extension NSApplication {
```

Comment:
This part wouldn't be necessary; the compiler generates stubs for the name as imported in Swift 2.